backend/test/test-schema.js

```diff
@@ -0,0 +1,21 @@
+const mongoose = require('mongoose');
+const Schema = mongoose.Schema;
+
+let TestSchema = new Schema({
+  name: {
+    type: String,
+    required: true
+  },
+  place: {
```

Comment:
is place referencing 'location'? should we use 'location' to be consistent?

Reply:
This scheme is for testing the db utilities function and really has nothing to do with the app itself.  We can change it if you want.